asv_bench/benchmarks/series_methods.py

```diff
@@ -141,7 +162,7 @@ def time_isin(self, dtypes, MaxNumber, series_type):
 
 class IsInLongSeriesValuesDominate:
     params = [
-        ["int64", "int32", "float64", "float32", "object"],
+        ["int64", "int32", "float64", "float32", "object", "Int64", "Float64"],
```

Comment:
This is causing failures with numpy 1.20

Int64 and Float64 now raise TypeError

https://github.com/pandas-dev/pandas/runs/1802727214